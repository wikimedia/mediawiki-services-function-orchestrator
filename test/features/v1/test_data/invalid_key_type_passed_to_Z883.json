{
	"Z1K1": "Z7",
	"Z7K1": "Z883",
	"Z883K1": "Z1",
	"Z883K2": "Z1"
}
