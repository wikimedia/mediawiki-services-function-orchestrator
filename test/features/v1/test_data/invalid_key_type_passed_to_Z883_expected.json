{
	"Z1K1": "Z5",
	"Z5K1": "Z516",
	"Z5K2": {
		"Z1K1": {
			"Z1K1": "Z7",
			"Z7K1": "Z885",
			"Z885K1": "Z516"
		},
		"Z516K1": "Z883K1",
		"Z516K2": {
			"Z1K1": "Z4",
			"Z4K1": "Z1",
			"Z4K2": [
				"Z3",
				{
					"Z1K1": "Z3",
					"Z3K1": "Z4",
					"Z3K2": "Z1K1",
					"Z3K3": {
						"Z1K1": "Z12",
						"Z12K1": [
							"Z11",
							{
								"Z1K1": "Z11",
								"Z11K1": "Z1002",
								"Z11K2": "type"
							}
						]
					}
				}
			],
			"Z4K3": "Z101"
		}
	}
}
